nupic/algorithms/anomaly.py

```diff
@@ -174,3 +199,41 @@ def __setstate__(self, state):
       self._movingAverage = None
     if not hasattr(self, '_binaryThreshold'):
       self._binaryThreshold = None
+    if not hasattr(self, '_computeFn'):
+      self._computeFn = computeRawAnomalyScore
+
+  ##################################################################
+  # these methods are alternative implementations of 'how to compute'
+  # anomaly score from the neural output (2 binary vectors)
+  ##################################################################
+def compute_In1D_Satisfied(activeColumns, prevPredictedColumns):
```

Comment:
I don't like moving the raw anomaly score here or changing the name to this (plus this name doesn't match camel case convention).